src/screens/RecoveryTransactionList/RecoveryTransactionListScreen.tsx

```diff
@@ -39,6 +48,11 @@ export class RecoveryTransactionListScreen extends PureComponent<Props, State> {
     selectedTransactions: [],
   };
 
+  componentDidMount() {
+    // sometimes we need nested this and will not touch performance
```

Comment:
we have the same issue with the transaction list on the dashboard screen and the fix for this was: https://nyxo.app/fixing-virtualizedlists-should-never-be-nested-inside-plain-scrollviews